config/supplementaries-registry.toml

```diff
@@ -32,15 +32,15 @@
 		sconce_lever = true
 		globe = true
 		hourglass = true
-		sack = true
+		sack = false
 		#WIP
 		flag = true
 		piston_launcher = true
 		cage = true
-		cog_block = true
+		cog_block = false
 		candle_holder = true
-		blackboard = true
-		safe = true
+		blackboard = false
+		safe = false

```

Comment:
Why did you disable blackboard and safe? :o

Reply:
They weren't mentioned in our initial discussion so I thought better to disable than leave enabled :) 

Other reasoning - for the safe - because other "secure/portable storage" options already exist (Mekanism/PneumaticCraft, others) and I didn't want to have a content overlap there. 

Blackboard[ seems to have a buggy UI](https://www.curseforge.com/minecraft/mc-mods/supplementaries/issues/14) (in my testing, only on certain GUI scales).

If you like them, we can flick them back on (and should wait for blackboard bug to be fixed, probably) :)

Reply:
I'd like the Safe re-enabled as I think people who aren't tech savvy still like safe-style storage :D